src/app/screens/BookDetail/components/CommentsList/components/Comment/styles.ts

```diff
@@ -0,0 +1,30 @@
+import { StyleSheet } from 'react-native';
+import { CHARCOAL } from '@constants/colors';
+
+const styles = StyleSheet.create({
+  commentContainer: {
+    display: 'flex',
```

Comment:
I think this is not necessary, the default (and only I think) display type in  RN is `'flex'`